slick.core.js

```diff
@@ -675,10 +726,251 @@
   function regexSanitizer(dirtyHtml) {
      return dirtyHtml.replace(/(\b)(on[a-z]+)(\s*)=|javascript:([^>]*)[^>]*|(<\s*)(\/*)script([<>]*).*(<\s*)(\/*)script(>*)|(&lt;)(\/*)(script|script defer)(.*)(&gt;|&gt;">)/gi, '');
   }
- 
-  // exports
-  $.extend(true, window, {
-    "Slick": {
+
+  // With help from https://youmightnotneedjquery.com/
+  function grep( elems, callback, invert ) {
+		var callbackInverse,
+			matches = [],
+			i = 0,
+			length = elems.length,
+			callbackExpect = !invert;
+
+		// Go through the array, only saving the items
+		// that pass the validator function
+		for ( ; i < length; i++ ) {
+			callbackInverse = !callback( elems[ i ], i );
+			if ( callbackInverse !== callbackExpect ) {
+				matches.push( elems[ i ] );
+			}
+		}
+
+		return matches;
+	}
+
+  function offset(el) {
+
+    box = el.getBoundingClientRect();
```

Comment:
`box` is missing the variable declaration `let` or `const` and because it is missing, it can throw this error that I received

```sh
referenceError: box is not defined
 ```